spec/Records.tex

```diff
@@ -752,7 +579,7 @@ \subsection{Default Comparison Operators}
 \index{classes!comparison}
 \index{records!comparison}
 
-For records, the compiler will supply default comparison operators if they are
-not supplied by the user.  The compiler does not supply default comparison
-operators for classes.
-
+For records, the compiler will supply default comparison operators if
+they are not supplied by the user. In contrast, the compiler only
+provides \chpl{==} and \chpl{!=} for classes and these perform pointer
+comparison.
```

Comment:
Instead of the second sentence, let's write this:

```
In contrast, the user cannot redefine \chpl{==} and \chpl{!=} for classes.
The default comparison operators for a record examine
the arguments' fields, while the comparison operators
for classes check whether the l.h.s. and r.h.s. refer
to the same class instance or are both \chpl{nil}.
```